test/unit/std/concept/core_language_test.cpp

```diff
@@ -78,66 +77,58 @@ TEST(explicitly_convertible_to_concept, basic)
 
 TEST(convertible_to_concept, basic)
 {
-    EXPECT_TRUE((convertible_to_concept<type_b,
-                                        type_c>));
-    EXPECT_TRUE((!convertible_to_concept<type_c,
-                                         type_b>));
-    EXPECT_TRUE((!convertible_to_concept<type_a,
-                                         type_c>));
+    EXPECT_TRUE((std::ConvertibleTo<type_b, type_c>));
+    EXPECT_TRUE((!std::ConvertibleTo<type_c, type_b>));
+    EXPECT_TRUE((!std::ConvertibleTo<type_a, type_c>));
 }
 
 TEST(common_reference_concept, basic)
 {
-    EXPECT_TRUE((common_reference_concept<int32_t, int16_t, int8_t>));
-    EXPECT_TRUE((!common_reference_concept<int32_t, int16_t, type_c>));
+    EXPECT_TRUE((std::CommonReference<int32_t, int16_t>));
+    EXPECT_TRUE((!std::CommonReference<int32_t, type_c>));
 }
 
 TEST(common_concept, basic)
 {
-    EXPECT_TRUE((common_concept<type_a,
-                                type_b>));
-    EXPECT_TRUE((!common_reference_concept<type_a,
-                                           type_c>));
+    EXPECT_TRUE((std::Common<type_a, type_b>));
+    EXPECT_TRUE((!std::Common<type_a, type_c>));
 }
 
 TEST(integral_concept, basic)
 {
-    EXPECT_TRUE((integral_concept<int>));
-    EXPECT_TRUE((!integral_concept<float>));
+    EXPECT_TRUE((std::Integral<int>));
+    EXPECT_TRUE((!std::Integral<float>));
 }
 
 TEST(signed_integral_concept, basic)
 {
-    EXPECT_TRUE((signed_integral_concept<int>));
-    EXPECT_TRUE((!signed_integral_concept<unsigned>));
+    EXPECT_TRUE((std::SignedIntegral<int>));
+    EXPECT_TRUE((!std::SignedIntegral<unsigned>));
 }
 
 TEST(unsigned_integral_concept, basic)
 {
-    EXPECT_TRUE((!unsigned_integral_concept<int>));
-    EXPECT_TRUE((unsigned_integral_concept<unsigned>));
+    EXPECT_TRUE((!std::UnsignedIntegral<int>));
+    EXPECT_TRUE((std::UnsignedIntegral<unsigned>));
 }
 
 TEST(assignable_concept, basic)
 {
-    EXPECT_TRUE((assignable_concept<type_a&,
-                                    type_a const &>));
-    EXPECT_TRUE((assignable_concept<type_c&,
-                                    type_b const &>));
-    EXPECT_TRUE((!assignable_concept<type_a&,
-                                     type_c&>));
+    EXPECT_TRUE((std::Assignable<type_a &, type_a const &>));
+    EXPECT_TRUE((std::Assignable<type_c &, type_b const &>));
+    EXPECT_TRUE((!std::Assignable<type_a &, type_c &>));
 }
 
 TEST(swappable_concept, basic)
 {
-    EXPECT_TRUE((swappable_concept<type_a>));
-    EXPECT_TRUE((swappable_concept<type_b>));
+    EXPECT_TRUE((std::Swappable<type_a>));
+    EXPECT_TRUE((std::Swappable<type_b>));
 }
 
 TEST(swappable_with_concept, basic)
 {
-    EXPECT_TRUE((swappable_with_concept<type_a&,
+    EXPECT_TRUE((std::SwappableWith<type_a&,
                                         type_a&>));
```

Comment:
put into one line with above

Reply:
yeah, I dunno, wo created those tests originally and why everything was weird.